bloom-housing/bloom
# 2723/listing availability listing browse

## Issue Overview

This PR addresses [#issue](https://github.com/bloom-housing/bloom/issues/2723)


## Description
This adds label on the public sites listing browse page to include the new listing availability

## How Can This Be Tested/Reviewed?
head to https://dev-partners-bloom.netlify.app/sign-in
create or edit a listing and set it to have its availability as `Available Units`
create or edit a second listing to have its availability set to `Open Waitlist`

head to https://dev-bloom.netlify.app/listings
head to the browse page 
verify that the labels are present like they should be

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
